monad.release-info

```diff
@@ -1,6 +0,0 @@
-(repo git "git@github.com:dleslie/monad-egg.git")
```

Comment:
Please un-remove the release-info file; this should remain for chicken 4 users.
We can create a chicken5.release-info file for the coop to reference.